MQ2Status.cpp

```diff
@@ -366,9 +365,9 @@ void StatusCmd(SPAWNINFO* pChar, char* szLine)
 				}
 				else {
 #endif !defined (ROF2EMU)
-					int altCurrency = AltCurrencyCheck(tempArg);
+					int altCurrency = GetCurrencyIDByName(tempArg.c_str());
```

Comment:
You can use NextArg here and save yourself a conversion.

Reply:
good eyeballs, fixed